Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -1315,6 +1317,18 @@ void MainWindow::ShowSkylanderPortal()
   m_skylander_window->activateWindow();
 }
 
+void MainWindow::ShowInfinityBase()
+{
+  if (!m_infinity_window)
+  {
+    m_infinity_window = new InfinityBaseWindow;
```

Comment:
Should this not be given a parent, like:

```suggestion
    m_infinity_window = new InfinityBaseWindow(this);
```

otherwise this could leak

Reply:
I copied this from the FIFO Window - which also is not given a parent. Should it also have this referenced? (The Skylander Portal Window has this missing as well)

Reply:
When I specify `(this)` as the parent, the window looks like this (on Mac at least): 

<img width="1301" alt="Screenshot 2023-04-11 at 11 42 38 AM" src="https://user-images.githubusercontent.com/28370004/231019905-48de7c9d-bfe7-4e5b-9d5e-cf9978c1fd86.png">

Is it appropriate to specify the parent as being null?